src/oop_ext/foundation/types_.py

```diff
@@ -10,6 +15,9 @@ class Method:
     """
 
 
+T = TypeVar("T")
+
+
 class Null:
     """
```

Comment:
I believe that class Null should have a `Generic[T]` since T is being used on `__iter__ ` and `__enter__`

Reply:
Thanks, actually realized we don't need a `T` variable here at all (see e663f2a).